frontend/packages/integration-tests-cypress/tests/favorite/favorite-option.cy.ts

```diff
@@ -0,0 +1,84 @@
+import { checkErrors } from '../../support';
+import { nav } from '../../views/nav';
+
+describe('Favorites', () => {
+  before(() => {
+    // clear any existing sessions
+    Cypress.session.clearAllSavedSessions();
+    cy.login();
```

Comment:
For individual file run we can include this change
```suggestion
    cy.login();
    guidedTour.close();
```